tests/test_citus.py

```diff
@@ -19,111 +19,113 @@ class TestCitus(BaseTestPostgresql):
     def setUp(self):
         super(TestCitus, self).setUp()
         self.c = self.p.mpp_handler
+        handler = self.c._citus_database_handlers
```

Comment:
I feel that there are to many unnecessary changes in unit tests.
I spent 10 minutes splitting them up to test separately CitusHandler and CitusDatabaseHandler.
Will push it soon.

Reply:
Pushed, please pay attention that now there are conflicts because of that: https://github.com/patroni/patroni/commit/efdf26a835694ddafb1dd5f444dfec4929aecfd4